pages/login.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
+import Router from 'next/router'
+import { useUser } from '../lib/hooks'
+import LoginForm from '../components/LoginForm'
+import Layout from '../components/Layout'
+
+const Login = () => {
+    const user = useUser({ redirectTo: '/', redirectIfFound: true })
```

Comment:
```suggestion
  const user = useUser({ redirectTo: '/', redirectIfFound: true })
```
i should really add eslint to this project